themes/src/main/resources/theme/keycloak/login/theme.properties

```
@@ -1,7 +1,7 @@
 parent=base
 import=common/keycloak
 
-styles=css/login.css
+styles=css/login.css css/password-entropy.css
```

Perhaps we can add the styles to `login.css` instead? That saves us from needing to request another file.
```suggestion
styles=css/login.css
```